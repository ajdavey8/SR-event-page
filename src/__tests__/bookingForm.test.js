import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import BookingForm from '../components/bookingForm';

describe(BookingForm, () => {
    const wrapper = shallow(<BookingForm/>);

    it('renders and matches snapshot', () => {
        const tree = renderer
            .create(<BookingForm/>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('renders a checkbox for I need a room and is not selected', () => {
        const radioOption = wrapper.find('#need').length;
        expect(radioOption).toEqual(1);
    });
});