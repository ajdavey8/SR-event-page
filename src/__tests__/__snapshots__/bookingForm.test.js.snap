// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookingForm renders and matches snapshot 1`] = `
<div>
  <form>
    <h2>
      Reserve Your Place Now
    </h2>
    <input
      id="need"
      name="need"
      type="radio"
      value="need"
    />
    <label>
       I need a room 
      <b>
        Places available
      </b>
       
    </label>
  </form>
</div>
`;
